.github/workflows/tag-for-release.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: googleapis/release-please-action@v4
```

Comment:
Please [pin this to the latest SHA instead of using a version tag](https://blog.rafaelgss.dev/why-you-should-pin-actions-by-commit-hash).